tools/lxdclient/client.go

```diff
@@ -38,12 +39,10 @@ func (p *lxdLogProxy) render(msg string, ctx []interface{}) string {
 		result.WriteString(": ")
 	}
 
-	/* This is sort of a hack, but it's enforced in the LXD code itself as
-	 * well. LXD's logging framework forces us to pass things as "string"
-	 * for one argument and then a "context object" as the next argument.
-	 * So, we do some basic rendering here to make it look slightly less
-	 * ugly.
-	 */
+	// This is sort of a hack, but it's enforced in the LXD code itself as well.
```

Comment:
Was this just reformatting the comment? It's a bit confusing - probably better to leave it in a non-standard format than muddy the water with a formatting change in the middle of changing behaviour.